### PR TITLE
Branch company poc and create command integration

### LINKED ISSUE #117: Implement create command
Implement a `create` command to create a POC.

### LINKED ISSUE #119: Integrate Company and POC

### LINKED ISSUE #124: Create Test Cases for Create Command

### PR DESCRIPTION
**Pull request by jicsontoh**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CreateCommand.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.Company;
+import seedu.address.model.poc.Poc;
+
+/**
+ * Changes the remark of an existing person in the address book.
+ */
```
Comment (notle1706) [inline src/main/java/seedu/address/logic/commands/CreateCommand.java]: Perhaps the javadocs should be updated?

File: src/main/java/seedu/address/logic/commands/CreateCommand.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            + "Index "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "HP "
+            + PREFIX_EMAIL + "EMAIL "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " 2 "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_PHONE + "97814456 "
+            + PREFIX_EMAIL + "test@test.com "
+            + PREFIX_TAG + "friends "
```
Comment (notle1706) [inline src/main/java/seedu/address/logic/commands/CreateCommand.java]: Perhaps this tag can be changed to be more in context? eg. Main Poc

**Review by notle1706 — CHANGES_REQUESTED** [review]
Looks good, just a few small changes to be made.

File: src/main/java/seedu/address/logic/commands/CreateCommand.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.Company;
+import seedu.address.model.poc.Poc;
+
+/**
+ * Creates a new poc to be added to the specified company.
+ */
+public class CreateCommand extends Command {
```
Comment (ElginL) [inline src/main/java/seedu/address/logic/commands/CreateCommand.java:22]: A well written class that doesn't violate any of the code quality standards 👍

**Review by ElginL — APPROVED** [review]
Nice additions with clean code. LGTM

File: src/main/java/seedu/address/logic/commands/CreateCommand.java (hunk 3 above)
Comment (jicsontoh) [reply src/main/java/seedu/address/logic/commands/CreateCommand.java:22]: Thank you Elgin

**Review by notle1706 — APPROVED** [review]

**Pull request merged**